src/main.js

```diff
@@ -73,7 +73,7 @@ async function bootstrapApp() {
         await store.dispatch("messages/setMessages");
     }
     catch {
-        store.commit("introspection/setMaintenanceMode");
+        await router.replace('/error/500')
```

Comment:
This catch block is reached when `await store.dispatch("introspection/fetchParameters");` is called on line 71 of main.js. So, the result is that setting maintenance mode causes a 500 internal server error to be shown rather than a maintenance page.

Reply:
I'm running your branch on my local system where I have control over the server. Turning it off causes the 500 to show as expected. Setting `maintenance = false` and all is fine. But, when `maintenance = true` the result is a 500 error. 

It's not possible to demonstrate this with the Netlify build due to the ongoing problem there.

Reply:
ok thanks. let me see what I can do to fix it. tnx

Reply:
I've dealt with this problem now and will ask for a review shortly.